ui/src/app/job-details/common/attempt/attempt.component.css

```diff
@@ -0,0 +1,135 @@
+  text-align: left;
+  min-width: 280px;
+  max-width: 280px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: block;
+  color: rgba(0, 0, 0, 0.87);
+  padding-left: 20px;
+}
+
+:host-context(mat-accordion.mat-accordion.content) mat-expansion-panel.mat-expansion-panel .task-name {
+  min-width: 180px;
+  max-width: 180px;
+}
+
+mat-expansion-panel.mat-expansion-panel mat-panel-description.mat-expansion-panel-header-description {
+  display: -webkit-box;
+  flex-flow: row;
+  justify-content: end;
+  color: rgba(0, 0, 0, 0.87);
+}
+
+mat-expansion-panel.mat-expansion-panel .task-status,
+mat-expansion-panel.mat-expansion-panel .task-start,
+mat-expansion-panel.mat-expansion-panel .task-duration,
+mat-expansion-panel.mat-expansion-panel .task-inputs,
+mat-expansion-panel.mat-expansion-panel .task-outputs,
+mat-expansion-panel.mat-expansion-panel .task-attempts {
+  text-align: left;
+}
+
+mat-expansion-panel.mat-expansion-panel .task-start,
+mat-expansion-panel.mat-expansion-panel .task-links {
+  width: 150px;
+}
+
+:host-context(mat-accordion.mat-accordion.content) mat-expansion-panel.mat-expansion-panel .task-start,
+:host-context(mat-accordion.mat-accordion.content) mat-expansion-panel.mat-expansion-panel .task-links {
+  width: 140px;
+}
+
+mat-expansion-panel.mat-expansion-panel .task-status,
+mat-expansion-panel.mat-expansion-panel .task-duration,
+mat-expansion-panel.mat-expansion-panel .task-inputs,
+mat-expansion-panel.mat-expansion-panel .task-outputs,
+mat-expansion-panel.mat-expansion-panel .task-attempts {
+  width: 110px;
+}
+
+:host-context(mat-accordion.mat-accordion.content) mat-expansion-panel.mat-expansion-panel .task-status,
+:host-context(mat-accordion.mat-accordion.content) mat-expansion-panel.mat-expansion-panel .task-duration,
+:host-context(mat-accordion.mat-accordion.content) mat-expansion-panel.mat-expansion-panel .task-inputs,
+:host-context(mat-accordion.mat-accordion.content) mat-expansion-panel.mat-expansion-panel .task-outputs,
+:host-context(mat-accordion.mat-accordion.content) mat-expansion-panel.mat-expansion-panel .task-attempts {
+  width: 100px;
+}
+
+mat-expansion-panel.mat-expansion-panel .task-links,
+mat-expansion-panel.mat-expansion-panel .task-attempts {
+  text-align: right;
+}
+
+mat-expansion-panel.mat-expansion-panel .task-links {
+  overflow: visible !important;
+}
+
+mat-expansion-panel.mat-expansion-panel .mat-expansion-panel-header[aria-disabled='true'] {
+  padding-right: 32px;
+  color: rgba(0, 0, 0, 0.54);
+}
+
+mat-expansion-panel.mat-expansion-panel .task-inputs clr-icon,
+mat-expansion-panel.mat-expansion-panel .task-outputs clr-icon {
+  color: #5C912E;
+  margin-right: 1rem;
+}
+
+mat-expansion-panel.mat-expansion-panel .task-duration clr-icon {
+  padding-left: 5px;
+}
+
+clr-tooltip-content {
+  background-color: rgba(0, 0, 0, 0.65);
+  padding: 0.25rem 0.5rem;
```

Comment:
Nit: I'd use either `rem` or `px` for padding, but not both. (Above you use `px`)